recipes/qr-code-generator/all/conandata.yml

```diff
@@ -0,0 +1,18 @@
+sources:
+  "1.4.0":
```

Comment:
Are these 3 versions required by any other projects you are using? If not, maybe just packaging the most recent version is enough

Reply:
I am using 1.4.0 now in my projects.

Reply:
ok, it is fine then. thanks! 😄